### DISCUSSION TITLE
Horizontal Progress Bar

### DISCUSSION THREAD

**Discussion by Raikeb** [ooflet/Mi-Create#17] [Questions-EN] [answered]
Even after reading the documentation, I couldn't understand how to make a horizontal progress bar. Inside the tool I just found how to make the Circle Progress Bar.
What I want to do:
![image](https://github.com/ooflet/Mi-Create/assets/69227820/ffdd3bd7-bc73-4af1-b011-d2f709de56b7)

![image](https://github.com/ooflet/Mi-Create/assets/69227820/5044b817-8218-4f3b-818c-5d6dce362e85)

What I did:
![image](https://github.com/ooflet/Mi-Create/assets/69227820/fc5c0101-a3b0-4cc4-89e2-68de75a36a5c)

The only way I found, which I believe to be wrong, was to make a giant circle and tune it so that it looks as least crooked as possible.
But this is probably not the right way.
How can I do it?

**Comment by ooflet** [comment, marked as answer]
The only way to make progress bars is to use the lineProgress_ prefix (e.g lineProgress_battery) as the name of a circleprogress element. Through here, the Progress Bar X/Y coordinate is used to define the initial position and start/end angle is used to define the end position. This is done to keep compatibility with EasyFace, there is no way to make line progress on there. Documentation (unfortunately only in Russian) for this system is here https://github.com/m0tral/EasyFace/wiki/ProgressbarRU
